### DISCUSSION TITLE
Rendering Error in layout  - Call to Undefined  Method

### DISCUSSION THREAD

**Discussion by gmaroko** [Kunena/Kunena-Forum#8071] [Q&A]
Hi all,
I haven't used this platform but just noting an error on a platform that has been persistent for a couple months now
**Rendering Error in layout Topic/Row: Call to undefined method KunenaUser::getName(). Please enable debug mode for more information.**
Is the above error a known bug or its something happening after an update ?

**Comment by 810** [comment]
Can you say which version are your running and which kunena template.

**Reply by gmaroko** [reply]
Hi @810 , thanks for your quick response.
As earlier noted in the original post albeit shallowly, I am an end user and that is an error that has been persistent on a forum I often use. I haven't been able to contact the owners. I am unsure which version they're running.
I picked the error from [here](http://www.t24all.com/index.php/forum/t24-development-a-customization/15224-java-programming.html) (for your ref).

**Reply by 810** [reply]
ok, i think he need to update it to the latest version, then it should work again.

**Reply by gmaroko** [reply]
I would think so yeah. I'm still on the hunt for the site owners so I can notify them if at all they didn't bother to check whether the site is up or not. LOL
Thanks tho

**Reply by xillibit** [reply]
Hello, without the kunena version installed and kunena addons versions installed, we can't say more.

**Reply by gmaroko** [reply]
@xillibit thanks! I have already reached out to the site administrator - awaiting their response. I bet it has got something to do with a missed update. Thanks again.
